spec/features/mainstream_browsing_spec.rb

```diff
@@ -29,4 +30,46 @@
       expect(page).to have_selector(".gem-c-breadcrumbs")
     end
   end
+
+  context "for AB tests" do
+    scenario "AB testing of new browse templates" do
+      with_variant NewBrowse: "B" do
+        schemas = GovukSchemas::Example.find_all("mainstream_browse_page").shuffle
+
+        schemas.each do |content_item|
```

Comment:
Nice!

Reply:
I can't claim the glory for the shuffle, pinched it from a test above! 😂